packages/@okta/vuepress-site/docs/guides/sign-into-spa/main/vue/reqauthspecific.md

```diff
@@ -1,30 +1,21 @@
-Set the `requiresAuth` property to `true` in the optional `meta` object as part of the route definition.
+Set the `requiresAuth` property to `true` in the optional `meta` object as part of the route definition. Add `navigationGuard` function from `okta-vue` as a global guard to the router.
```

Comment:
I think the wording could be clearer here. Would "Add the `navigationGuard` function from `okta-vue` to the router as a global guard." still make sense?